kolibri/plugins/coach/serializers.py

```diff
@@ -68,12 +67,11 @@ def get_last_active(self, target_user):
 class ContentReportSerializer(serializers.ModelSerializer):
     progress = serializers.SerializerMethodField()
     last_active = serializers.SerializerMethodField()
-    parent = serializers.SerializerMethodField()
 
     class Meta:
         model = ContentNode
         fields = (
-            'pk', 'content_id', 'title', 'progress', 'kind', 'last_active', 'parent',
+            'pk', 'content_id', 'title', 'progress', 'kind', 'last_active',
```

Comment:
Is this not needed anymore? I remember something with the parent was needed for breadcrumbs